src/Http/Http.Abstractions/src/Routing/RouteValueDictionary.cs

```diff
@@ -100,7 +100,8 @@ public RouteValueDictionary()
     /// property names are keys, and property values are the values, and copied into the dictionary.
     /// Only public instance non-index properties are considered.
     /// </remarks>
-    [RequiresUnreferencedCode("This constructor may perform reflection on the specificed value which may be trimmed if not referenced directly. Consider using a different overload to avoid this issue.")]
+    [RequiresUnreferencedCode("This constructor may perform reflection on the specificed value which may be trimmed if not referenced directly. Consider using a different overload to avoid this issue.")] // PropertyStorage
+    [RequiresDynamicCode("This constructor may perform reflection on the specificed value which may be trimmed if not referenced directly. Consider using a different overload to avoid this issue.")] // PropertyStorage
```

Comment:
What do the `PropertyStorage` comments at the end of the line mean?

Reply:
It's an indication of what is causing this warning in the method.